lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const stackPacks = require('@lighthouse/stack-packs');
+
+const stackPacksToInclude = [{
+  packId: 'wordpress',
+  npmOrIconName: 'wordpress',
+  requiredStacks: ['js:wordpress']
+}];
+
+/**
+ * @param {LH.Artifacts} artifacts
+ * @return {Array<LH.StackPack>}
+ */
+function getStackPacks(artifacts) {
+  /** @type {Array<LH.StackPack>} */
+  const packs = [];
+
+  artifacts.Stacks.forEach(pageStack => {
```

Comment:
this will cause an error if "best-practices" is not run (only BP audits run the stacks gatherer)

Reply:
Aye.

Seems like with stackpacks we'll have to move the Stack gatherer to be a base artifact. Running here makes sense to me:
https://github.com/GoogleChrome/lighthouse/blob/584bad7c767132010908275a94862d375347506a/lighthouse-core/gather/gather-runner.js#L482

Reply:
What's a base artifact? :p it's ran nonetheless?

Reply:
basically yes.